studio/components/layouts/SQLEditorLayout/QueryItem.tsx

```diff
@@ -200,44 +203,44 @@ const QueryItemActions = observer(({ tabInfo, activeId }: QueryItemActionsProps)
   return (
     <div className="group [div&>button[data-state='open']>span]:text-scale-900">
       {IS_PLATFORM ? (
-        <Dropdown
-          side="bottom"
-          align="end"
-          overlay={
-            <>
-              <Dropdown.Item onClick={onClickRename} icon={<IconEdit2 size="tiny" />}>
-                Rename query
-              </Dropdown.Item>
-              {sharedSnippetsFeature && visibility === 'user' && canCreateSQLSnippet && (
-                <Dropdown.Item onClick={onClickShare} icon={<IconShare size="tiny" />}>
-                  Share query
-                </Dropdown.Item>
-              )}
-              {sharedSnippetsFeature && visibility === 'project' && canCreateSQLSnippet && (
-                <Dropdown.Item onClick={createPersonalCopy} icon={<IconCopy size="tiny" />}>
-                  Create a personal copy
-                </Dropdown.Item>
+        <DropdownMenu_Shadcn_>
+          <DropdownMenuTrigger_Shadcn_>
+            <span
+              className={clsx(
+                'p-0.5 rounded-md',
+                isActive
+                  ? 'text-scale-1100 hover:bg-scale-800'
+                  : 'text-scale-300 dark:text-scale-200 hover:bg-scale-500 group-hover:text-scale-1100'
               )}
-              <>
-                <Dropdown.Separator />
-                <Dropdown.Item onClick={onClickDelete} icon={<IconTrash size="tiny" />}>
-                  Delete query
-                </Dropdown.Item>
-              </>
-            </>
-          }
-        >
-          <span
-            className={clsx(
-              'p-0.5 rounded-md',
-              isActive
-                ? 'text-scale-1100 hover:bg-scale-800'
-                : 'text-scale-300 dark:text-scale-200 hover:bg-scale-500 group-hover:text-scale-1100'
+            >
+              <IconChevronDown size="tiny" strokeWidth={2} />
+            </span>
+          </DropdownMenuTrigger_Shadcn_>
+          <DropdownMenuContent_Shadcn_ side="bottom" align="end">
+            <DropdownMenuItem_Shadcn_ onClick={onClickRename}>
+              <IconEdit2 size="tiny" />
+              <p className="text-scale-1200 text-sm">Rename query</p>
+            </DropdownMenuItem_Shadcn_>
+            {sharedSnippetsFeature && visibility === 'user' && canCreateSQLSnippet && (
+              <DropdownMenuItem_Shadcn_ onClick={onClickShare}>
+                <IconShare size="tiny" />
+                <p className="text-scale-1200 text-sm">Share query</p>
+              </DropdownMenuItem_Shadcn_>
             )}
-          >
-            <IconChevronDown size="tiny" strokeWidth={2} />
-          </span>
-        </Dropdown>
+            {sharedSnippetsFeature && visibility === 'project' && canCreateSQLSnippet && (
+              <DropdownMenuItem_Shadcn_ onClick={createPersonalCopy}>
+                <p className="text-scale-1200 text-sm">Create a personal copy</p>
```

Comment:
This guy is looking a little lonely without an icon 🥲
<img width="286" alt="Screenshot 2023-09-15 at 17 13 29" src="https://github.com/supabase/supabase/assets/10985857/2470b934-a89e-4574-ad91-38d115807690">

Reply:
Probably should say "Duplicate personal copy".
'Duplicate' being the important word since that is what is happening.
then use a copy icon.

Reply:
can do a follow up PR though